.changeset/tasty-buses-jog.md

```diff
@@ -0,0 +1,127 @@
+---
+'@keystone-next/keystone': major
+---
+
+The names of one-sided and two-sided, many-many relationships has been shortened.  Two-sided many-many relationship names contain only the left-hand side names now; and the `_many` suffix has been dropped from one-sided many-many relationships.
+
+This reduces the probability that you will exceed [PostgreSQL's 63 character limit for identifiers](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS) with typical usage.
+
+This is a breaking change. 
+
+There are two ways to update:
+
+1. Set `experimental.useLegacyManyRelationNames: true` at the root of your config. This will make Keystone revert to the old relation names.  This feature will be removed in a future release.
```

Comment:
```suggestion
1. Set `experimental.useLegacyManyRelationNames: true` at the root of your config. This will make Keystone revert to the old relation names. This feature will be removed in a future release.
```